compoments/report-repair/contact-details.js

```diff
@@ -50,12 +52,8 @@ const ContactDetails = ({handleChange, values}) => {
   const beforeButton =  (
     <Details summary="I have neither a mobile number nor an email address" testid="no-applicable-contact-options-prompt">
       <div data-testid='no-applicable-contact-options-info'>
-        Please call 01522 873333 to report your repair during the office hours below
-        <li>Monday 9am - 5pm</li>
-        <li>Tuesday 9am - 5pm</li>
-        <li>Wednesday 9am - 4.30pm</li>
-        <li>Thursday 10am - 5pm</li>
-        <li>Friday 9am - 4.30pm</li>
+        Please call {customerServicesTelephoneNumber} to report your repair during the office hours below
```

Comment:
Nitpick: Add an colon at the end of this line
```suggestion
        Please call {customerServicesTelephoneNumber} to report your repair during the office hours below:
```